docs/insights/metrics.md

```diff
@@ -21,3 +33,12 @@ Failure rate is calculated by taking the total number of deployments, and dividi
 ## Mean time to recovery
 
 For a given failed deployment, its time to recovery is the time period from its completion time, to the completion time of the next successful deployment within the same scope.
+
+This chart shows the simplest example of the time to recovery, where a failed deployment is successfully redeployed straight afterwards.
+![Mean Time To Recovery Example 0](images/mean-time-to-recovery-example-0.png)
+
+Time to recovery can span across multiple releases too.  In this scenario, the deployment of release 1.2 has failed, but release 1.3 was deployed instead of re-trying 1.2.  The time to recovery is the time from the completion of the failed deployment of 1.2, to the completion of the successful deployment of 1.3.
+![Mean Time To Recovery Example 1](images/mean-time-to-recovery-example-1.png)
+
+Scope is taken into consideration when calculating time to recovery.  In the example below, you can see that the time to recovery for the Test environment is shorter than that of the Production environment, because the earliest failure was for the Production environment.
+![Mean Time To Recovery Example 2](images/mean-time-to-recovery-example-2.png)
```

Comment:
```suggestion

This chart shows the simplest example of the time to recovery, where a failed deployment is successfully redeployed straight afterward.
![Mean Time To Recovery Example 0](images/mean-time-to-recovery-example-0.png)

Time to recovery can span across multiple releases too.  In this scenario, the deployment of release 1.2 failed, but release 1.3 was deployed instead of re-trying 1.2.  The time to recovery is the time from the completion of the failed deployment of 1.2 to the completion of the successful deployment of 1.3.
![Mean Time To Recovery Example 1](images/mean-time-to-recovery-example-1.png)

Scope is taken into consideration when calculating time to recovery.  In the example below, you can see that the time to recovery for the Test environment is shorter than that of the Production environment because the earliest failure was for the Production environment.
![Mean Time To Recovery Example 2](images/mean-time-to-recovery-example-2.png)
```